device/src/esp32_run.rs

```diff
@@ -587,6 +587,22 @@ where
                                     ui::Prompt::ConfirmLoadBackup(share_backup),
                                 ));
                             }
+                            DeviceToUserMessage::VerifyAddress { spk, bip32_path } => {
+                                let address =
+                                    bitcoin::Address::from_script(&spk.spk(), BITCOIN_NETWORK)
```

Comment:
Little  hacky passing the spk around instead of the address so we can use the const easily here.